src/quadrature/quadrature_nodal_3D.C

```diff
@@ -122,6 +122,82 @@ void QNodal::init_3D(const ElemType, unsigned int)
         return;
       }
 
+    case PRISM20:
+      {
+        _points =
+          {
+            Point(0.,0.,-1), Point(+1,0.,-1), Point(0.,+1,-1),
+            Point(0.,0.,+1), Point(+1,0.,+1), Point(0.,+1,+1),
+            Point(.5,0.,-1), Point(.5,.5,-1), Point(0.,.5,-1),
+            Point(0.,0.,0.), Point(+1,0.,0.), Point(0.,+1,0.),
+            Point(.5,0.,+1), Point(.5,.5,+1), Point(0.,.5,+1),
+            Point(.5,0.,0.), Point(.5,.5,0.), Point(0.,.5,0.),
+            Point(1/Real(3),1/Real(3),-1), Point(1/Real(3),1/Real(3),+1)
+          };
+
+        // Symmetry gives us identical weights on vertices, triangle
+        // edges, square edges, triangle faces, square faces; then we
+        // have the midnode.  Solving for weights which exactly
+        // integrate cubics and xi^2*zeta^2 would give a unique answer
+        // ... with wse=0 and wte<0.  See Quad8 in
+        // quadrature_nodal_2D.C for discussion of this problem.
+        //
+        // Dropping the xi^2 zeta^2 constraint gives us a rank-1 null
+        // space to play with ... but adding anything from that null
+        // space to push wte closer to positive just pushes wse
+        // negative.
+        //
+        // Dropping exact integration of xi^3 type terms gives us a
+        // rank-2 null space.  I just found the max(min(weight)) from
+        // that solution space using Octave.  Someone who cares more
+        // than me might want to repeat this exercise with better than
+        // double precision...
```

Comment:
I'll also check what "nodal quadrature mass matrix optimization" approach for this case gives when I get a chance.

Reply:
That'd be great!  I didn't quite hit the uncanny valley then, huh?